changelog.asc

```diff
@@ -2,6 +2,21 @@
 :sectanchors:
 = AnkiDroid Changelog
 
+== Version 2.16.3 (20230807)
+* We meet Play Store requirements again, so we can release fixes again!
+* This is a stability release for the recent 2.16 series launch
+* https://opencollective.com/ankidroid[🤜🤛 Thank you! Your support makes the fixes happen!]
+* Fix inconsistent italics between Anki / AnkiDroid
+* Fix crash in certain cases adding images
+* Fix display of saved card searches
+* Add ability to map "reschedule" to a gesture
+* Fix crash exporting decks with '/' in deck name
+* Fix crash displaying whiteboard menu
+* Allow long preference titles to wrap
+* Allow 3rd party cookies in webview to match Anki behavior
+* Improve "add shortcut" icon
```

Comment:
```suggestion
* Improve "add" shortcut icon
```

Reply:
I removed the quotes entirely based on this suggestion